README.md

```diff
@@ -1 +1,38 @@
-# go-clang-phoenix-bootstrap
+# go-clang-phoenix-bootstrap [![GoDoc](https://godoc.org/github.com/zimmski/go-clang-phoenix-bootstrap?status.png)](https://godoc.org/github.com/zimmski/go-clang-phoenix-bootstrap) [![Build Status](https://travis-ci.org/zimmski/go-clang-phoenix-bootstrap.svg?branch=master)](https://travis-ci.org/zimmski/go-clang-phoenix-bootstrap) [![Coverage Status](https://coveralls.io/repos/zimmski/go-clang-phoenix-bootstrap-gen/badge.png?branch=master)](https://coveralls.io/r/zimmski/go-clang-phoenix-bootstrap?branch=master)
+
+Native Go bindings for the C API of Clang.
+
+#Usage
+
+There is no direct way to use this repo other than for bootstrapping new go-clang-bindings. 
```

Comment:
Please rewrite this section so that we can just copy this README for all version repositories.